.github/workflows/lint.yml

```
@@ -0,0 +1,35 @@
+# SPDX-FileCopyrightText: 2024 The PFDL VS Code Extension Contributors
+# SPDX-License-Identifier: MIT
+
+name: Lint
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
```

```suggestion
  push:
    branches:
      - main
```
You don't want to run this on every push in every branch.

---

As far as I understand @ptrdtznr, this is exactly the behaviour we want (linting on every push / branch)

---

No, this is definitely not what you want ad we only have limited CI resources. Lint on push to main and on PRs should be sufficient. Do you really want to get an email on every commit your pushing to a feature / development branch?

---

Okay I see.